modules/administration/pages/backup-restore.adoc

```diff
@@ -415,8 +443,10 @@ public.log                            | 32 kB
 It is possible to move the database to another location.
 For example, if database storage space is running low.
 
-
-
+////
```

Comment:
I don't think we should keep this procedure anymore. It can even cause problems with SELinux label in the way it's described in here.